### ISSUE TITLE
Fresh Install leaves Sudo commands open without password

### ISSUE THREAD

**Issue by kronikpillow** [LukeSmithxyz/LARBS#517]
upon fresh install, any sudo command can be executed without password due to the LARBS installer leaving a residual file left inside of /etc/sudoers.d from the installer self, that is meant to be temporary named /etc/sudoers.d/larbs-temp ...

im sure this isn't the intended behavior and that who ever added that line, forgot to add a line that removes it after installation is complete :)

**Comment by LukeSmithxyz** [comment]
The line right above the line that creates the file traps a removal command for it. Did your script end in an unexpected way?

**Comment by kronikpillow** [comment]
Nope, it ended perfectly to the end, with all packages installed, except that the temp permissions didnt get removed, all 4 attempts this week ... Aka 4 reinstalls, same result

**Comment by exlumine** [comment]
So deleting `/etc/sudoers.d/larbs-temp` leads back to normal behavior.
